cwm_worker_operator/domains_config.py

```diff
@@ -448,11 +452,12 @@ def get_cwm_api_volume_config(self, metrics=None, force_update=False, hostname=N
                     metrics.cwm_api_volume_config_error_from_api(worker_id or 'missing', start_time)
             return VolumeConfig(volume_config, self, request_hostname=hostname, is_data_from_cache=False, request_worker_id=worker_id)
         else:
+            volume_config = json.loads(val)
+            if not worker_id:
+                worker_id = volume_config['instanceId']
```

Comment:
you should not parse the volume config here, the parsing logic may change and should be only handled internally in the VolumeConfig class
instead, you should set volume config to a variable and then use volume_config.id